tests/foreman/cli/test_activationkey.py

```diff
@@ -8,7 +8,7 @@
 
 :CaseLevel: Acceptance
```

Comment:
The same comment as I left for `CaseLevel` in `tests/foreman/api/test_activationkey.py` - it probably makes more sense to change it to `Component`.